pkg/cluster/cluster.go

```diff
@@ -59,7 +59,7 @@ type Node struct {
 
 // String ...
 func (node Node) String() string {
-	return fmt.Sprintf("%s %s:%d", node.PodName, node.IP, node.Port)
+	return fmt.Sprintf("node[ns=%s,ip=%s:%d]", node.PodName, node.IP, node.Port)
```

Comment:
ns=podname?

Reply:
Thanks and resolved on 504f4fdfe0c72ac9fc445a264a5d3469c991b9d9.